### PR TITLE
Long time no see

### PR DESCRIPTION
**Pull request by crishaddad**
Hi there; sorry but in my PR back in December, I got a couple of app filters wrong.

I've merged in the current master, and updated the appfilter.xml again; hopefully I've got these right this time, but note that for both Transdroid/Transdrone and Power Ampache, I am having a little difficulty getting the actions as they crash out the "Application Info" app, so I have had to pull these from reading the manifests directly and making educated guesses. Apologies if they are [still] not right.

### CODE REVIEW

File: app/src/main/res/xml/appfilter.xml [hunk 1]
```diff
@@ -2679,6 +2682,10 @@
     <!-- Transdrone -->
     <item component="ComponentInfo{org.transdroid.lite/org.transdroid.core.gui.TorrentsActivity_}" drawable="transdrone" />
 
+    <!-- Transdroid -->
+    <item component="ComponentInfo{org.transdroid.search/org.transdroid.search.gui.SettingsActivity}" drawable="transdroid"/>
+    <item component="ComponentInfo{org.transdroid.full/org.transdroid.core.gui.TorrentsActivity_}" drawable="transdroid"/>
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml]: It looks like Transdroid is already added five lines above this section, are those incorrect?

**Comment by crishaddad** [comment]
Sorry, this seems to be a duplicate; I've combined the two sections into a single section.

File: app/src/main/res/xml/appfilter.xml [hunk 2]
```diff
@@ -2679,13 +2682,14 @@
...
-    <item component="ComponentInfo{org.transdroid.search/org.transdroid.search.gui.SettingsActivity}" drawable="transdroid" />
-    <item component="ComponentInfo{org.transdroid.full/org.transdroid.core.gui.TorrentsActivity}" drawable="transdroid" />
-
     <!-- Transdrone -->
     <item component="ComponentInfo{org.transdroid.lite/org.transdroid.core.gui.TorrentsActivity_}" drawable="transdrone" />
 
+    <!-- Transdroid -->
+    <item component="ComponentInfo{org.transdroid.search/org.transdroid.search.gui.SettingsActivity}" drawable="transdroid"/>
+    <item component="ComponentInfo{org.transdroid.full/org.transdroid.core.gui.TorrentsActivity}" drawable="transdroid"/>
+    <item component="ComponentInfo{org.transdroid.full/org.transdroid.core.gui.TorrentsActivity_}" drawable="transdroid"/>
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml]: I'm not sure underscores are even allowed for class names, are you sure this was right?

Comment (crishaddad) [reply app/src/main/res/xml/appfilter.xml]: >  I'm not sure underscores are even allowed for class names

There are plenty of other apps listed in the manifest that use underscores?!

I believe that "SettingsActivity" and "TorrentsActivity_" are the actual correct 2 classes, but I _think_ "TorrentsActivity" was already added by someone else, so I didn't want to remove it, and the fact that there seems to be 2 different naming schenes (i.e. with and without underscore) confuses me a little .  The app manifest is [here](https://github.com/erickok/transdroid/blob/master/app/src/main/AndroidManifest.xml); perhaps, someone can double check my work?

Comment (crishaddad) [reply app/src/main/res/xml/appfilter.xml]: It's also confused by the fact there are multiple versions of the app, and also a search plugin that appears as a seperate icon alongside. Currently, for me, the search plugin has the Frost icon, but the main app does not.

Comment (dkanada) [reply app/src/main/res/xml/appfilter.xml]: I'm not sure about why certain ones aren't applying, that's weird.

**Pull request merged**